pegasus/sites.v3/hourofcode.com/views/announcements.haml

```diff
@@ -1,19 +1,32 @@
-:ruby
-  providers = []
+-# Commenting out the Robotics promo code below for potential future use
```

Comment:
I'd be in favor of putting this in another view and not using it, or deleting it (unless you got specific instructions that we'll need to again) –– not blocking though

Reply:
I like this idea! Just made a separate view: [`announcements_robotics.haml`](https://github.com/code-dot-org/code-dot-org/blob/15975be40599155e3a512f42ffd378803ea139d1/pegasus/sites.v3/hourofcode.com/views/announcements_robotics.haml)